controllers/operatorpolicy_status.go

```diff
@@ -427,6 +439,13 @@ var noCSVCond = metav1.Condition{
 	Message: "A relevant installed ClusterServiceVersion could not be found",
 }
 
+var noDeploymentsCond = metav1.Condition{
```

Comment:
I wasn't expecting to see this in this PR's diff, it might indicate a merge conflict gone wrong?